index.html

```diff
@@ -37,63 +39,59 @@
 		<section class="news">
 			<div class="newsvideo">
 				<iframe width="560" height="315" src="https://www.youtube.com/embed/AaLSXrRSQRw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Maybe you could add these styles in css file. 
for example: `width="560" height="315"`